website/source/guides/operations/vault-integration/index.html.md

```diff
@@ -1,71 +1,124 @@
 ---
 layout: "guides"
-page_title: "Vault Integration"
+page_title: "Vault Integration and Retrieving Dynamic Secrets"
 sidebar_current: "guides-operations-vault-integration"
 description: |-
-  Learn how to integrate Nomad with HashiCorp Vault and retrieve Vault tokens for
-  tasks.
+  Learn how to deploy an application in Nomad and retrieve dynamic credentials
+  by integrating with Vault.
 ---
 
-# Vault Integration
+# Vault Integration and Retrieving Dynamic Secrets
 
-Many workloads require access to tokens, passwords, certificates, API keys, and
-other secrets. To enable secure, auditable and easy access to your secrets,
-Nomad integrates with HashiCorp's [Vault][]. Nomad servers and clients
-coordinate with Vault to derive a Vault token that has access to only the Vault
-policies the tasks needs. Nomad clients make the token available to the task and
-handle the tokens renewal. Further, Nomad's [`template` block][template] can
-retrieve secrets from Vault making it easier than ever to secure your
-infrastructure.
+Nomad integrates seamlessly with [Vault][vault] and allows your application to
+retrieve dynamic credentials for various tasks. In this guide, you will deploy a
+web application that needs to authenticate against [PostgreSQL][postgresql] to
+display data from a table to the user.
 
-Note that in order to use Vault with Nomad, you will need to configure and
-install Vault separately from Nomad. Nomad does not run Vault for you.
+## Reference Material
 
--> **Note:** Vault integration requires Vault version 0.6.2 or higher.
+- [Vault Integration][vault-integration]
```

Comment:
We may want to label this link "Vault Integration Docs Page" or similar here to differentiate it from this new Vault Integration guide/tutorial page.